frontend/pages/shelter/login.tsx

```diff
@@ -15,8 +16,10 @@ const ShelterLogin = () => {
 			const axios = createAxiosInstance();
 			const {
 				data: { payload },
-			} = await axios.post("/api/login", values);
-			window.location.assign("/shelter/home");
+			} = await axios.post<LoginResponse>("/api/login", values);
+			if (payload.shelterId) {
+				window.location.assign("/shelter/home");
+			}
```

Comment:
might not be in the scope of your PR, but what's supposed to happen when there's no shelter id?

Reply:
not sure yet. we need to decide on the home page for adopters.